platform/android/app/src/main/kotlin/coop/polypoly/polypod/features/FeatureStorage.kt

```diff
@@ -28,7 +28,10 @@ object FeatureStorage {
 
         copyFeatureCategories(context)
         copyFeatures(context)
-        categories = Core.loadFeatureCategories(getFeaturesDir(context).path)
+        categories = Core.loadFeatureCategories(
+            getFeaturesDir(context).path,
+            emptyList()
```

Comment:
Didn't feel like adding a TODO comment, but this is basically what I'll use to run the tests in the same way as on iOS on Android